tools/ci_build/github/apple/test_apple_packages.py

```diff
@@ -206,6 +206,13 @@ def parse_args():
         help="Prepare the test project only, without running the tests",
     )
 
+    parser.add_argument(
+        "--skip_macos_test",
+        action="store_true",
+        help="Skip macos platform tests. Specify this argument when build targets only contain ios archs. "
+        "(Currently only applies to the build using default_full_ios_framework_build_settings.json config.)",
```

Comment:
nit: I'd leave this line out as it's too easy to become outdated. it's simple enough to search the code for all occurrences of `--skip_macos_test` if one wants to see where it applies.

Reply:
ok, sure. will remove the content in the parenthesis.